### DISCUSSION TITLE
Kgt Idle now has a discord, for some reason

### DISCUSSION THREAD

**Discussion by geckoking7** [geckoking7/kgt_idle#22] [Announcements]
Join here ---> https://discord.com/invite/7k8YQ3twT2

**Comment by unc00rdinated** [comment]
sorry i don't have discord, otherwise i would want to join and send memes

**Reply by geckoking7** [reply]
It's fine

**Reply by ExplodingHairyPotatoCat** [reply]
I mean you can always make one
(I'm not sure why but this sounds weird to me)
